src/launcher/launcher_main.cpp

```diff
@@ -499,6 +508,7 @@ void mouse_click() {
 		return;
 	case ui_obj_play_game:
 		if(file_is_ready.load(std::memory_order::memory_order_acquire) && !selected_scenario_file.empty()) {
+			save_playername();
```

Comment:
shouldn't you save it when it is changed, not every time the game is started?

Reply:
its easier this way - people often change their name before joining a game

Reply:
It also means you are going to be repeatedly saving it when nothing has changed. It also means that you will be silently discarding changes unless someone launches a game after making them.

Reply:
ok fine